api/src/entities/Anime.ts

```diff
@@ -0,0 +1,177 @@
+import {
+    Entity,
+    BaseEntity,
+    PrimaryGeneratedColumn,
+    Column,
+    CreateDateColumn,
+    UpdateDateColumn,
+    ManyToOne,
+    ManyToMany,
+    Unique,
+    JoinTable
+} from 'typeorm';
+import Character from './Character.js';
+import Studio from './Studio.js';
+
+export enum AnimeStatus {
+    AIRING = "airing",
+    UPCOMING = "upcoming",
+    FINISHED = "finished"
+};
+
+export enum AnimeFormat {
+    TELEVISION = "TV",
+    TELEVISION_SHORT = "TV short",
+    MOVIE = "movie",
+    SPECIAL = "special",
+    ORIGINAL_VIDEO_ANIMATION = "ova",
+    ORIGINAL_NET_ANIMATION = "ona",
+    MUSIC = "music"
+};
+
+export enum AnimeSeason {
+    SPRING = "spring",
+    SUMMER = "summer",
+    FALL = "fall",
+    WINTER = "winter"
+};
+
+export enum AnimeSource {
+    LIGHT_NOVEL = "light novel",
+    WEB_NOVEL = "web novel",
+    NOVEL = "novel",
+    ANIME = "anime",
+    VISUAL_NOVEL = "visual novel",
+    VIDEO_GAME = "video game",
+    DOUJINSHI = "doujinshi",
+    COMIC = "comic",
+    MANGA = "manga",
+    LIVE_ACTION = "live action",
+    GAME = "game",
+    MULTIMEDIA_PROJECT = "multimedia project",
+    PICTURE_BOOK = "picture book",
+    OTHER = "other"
+};
+
+@Entity('anime')
+@Unique(['title_english'])
+@Unique(['title_romaji'])
+@Unique(['title_native'])
+class Anime extends BaseEntity {
+
+    constructor(
+        id: number,
+        title_english: string,
+        title_romaji: string,
+        title_native: string,
+        description: string,
+        format: AnimeFormat,
+        episodes: number,
+        episode_duration: number,
+        source: AnimeSource,
+        status: AnimeStatus,
+        release_year: number,
+        start_date: Date,
```

Comment:
It depends on how we want to handle rating. We can cross reference other anime apis or sites like MyAnimeList or AniList every so often (maybe once a day or week) and we can store that. Or we can allow users to rate anime. If we do the second option, we'd have to use a separate table. Downside is we have less of a sample size, so ratings from our end would be inaccurate.

Since ratings might be a little more complex, it would be something we address down the line as its own issue